server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/Navigation.java

```diff
@@ -258,6 +258,8 @@ public SelenideElement getErrorMessage() {
     return Selenide.$("#error");
   }
 
+  public EmailAlreadyExistsPage asEmailAlreadyExistsPage() { return new EmailAlreadyExistsPage(); }
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this line start at column 5. ([squid:IndentationCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-1161%2Fallow_auth_of_same_email&issues=AWFSC5wUsS4rYBZqFLzM&open=AWFSC5wUsS4rYBZqFLzM)

[ISSUE_KEY]: AWFSC5wUsS4rYBZqFLzM